LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 Istio Authors
+   Copyright 2018 Istio Authors
```

Comment:
You do not need to update this license but only the new license

Reply:
done